InternshipApp/InternshipApp/Screens/EditProfileScreenView/EditProfileView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  EditProfileView.swift
+//  InternshipApp
+//
+//  Created by Sead Mašetić on 13. 8. 2024..
+//
+
+import SwiftUI
+import PhotosUI
+
+struct EditProfileView: View {
+    
+    @State private var localName: String
+    @State private var localEmail: String
+    @State private var localImage: UIImage?
```

Comment:
All properties should be inside it's respective viewmodel, and need to be accessed from there :)